WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingTaskView.swift

```diff
@@ -40,11 +40,25 @@ struct StoreOnboardingTaskView: View {
                         // Task labels
                         VStack(alignment: .leading, spacing: Layout.verticalSpacing) {
                             Spacer().frame(height: Layout.spacerHeight)
-                            // Task title.
-                            Text(viewModel.title)
-                                .headlineStyle()
-                                .multilineTextAlignment(.leading)
-                                .redacted(reason: isRedacted ? .placeholder : [])
+
+                            HStack {
+                                // Task title.
+                                Text(viewModel.title)
+                                    .headlineStyle()
+                                    .multilineTextAlignment(.leading)
+                                    .redacted(reason: isRedacted ? .placeholder : [])
+
+                                // PRIVATE label.
+                                Text(Localization.privateLabel)
+                                    .font(.caption)
+                                    .fontWeight(.bold)
+                                    .foregroundColor(Color(uiColor: .accent))
+                                    .multilineTextAlignment(.leading)
```

Comment:
Nit: this is unlikely to have multiple line, maybe this is not needed?

Reply:
Done in 1c9edb472517ae603c927e7f1d87c5f1d33b4e65